tests/integration/test_sysctl.py

```diff
@@ -8,7 +8,7 @@
 
 from charms.operator_libs_linux.v0 import sysctl
 
-EXPECTED_MERGED_RESULT = """# This config file was produced by sysctl lib v0.2
+EXPECTED_MERGED_RESULT = """# This config file was produced by sysctl lib v0.3
```

Comment:
Can we use `v{LIBAPI}.{LIBPATCH}` here (and below), to avoid having to update this every update?

Reply:
Yes, that is much better.